PatientMonitoringSystem/PatientMonitoringSystem/Controllers/ModuleRowController.cs

```diff
@@ -49,10 +49,25 @@ public void UpdateMaxValue(int value)
 			moduleRowView.UpdateMaxValue(value);
 		}
 
+		public bool CheckValueBreached(int value) {
```

Comment:
@harrytpstudent I would have put this in the "backend" because it is business logic and not tied directly to a particular UI implementation.